juniper_official/Solutions/Krt-async-stuck/krtasync.playbook

```diff
@@ -0,0 +1,157 @@
+/ *
```

Comment:
480
Entering configuration mode
The configuration has been changed but not committed
|/rules/juniper_official/Solutions/Krt-async-stuck/krtasync.playbook:1:(1) syntax error: /
/rules/juniper_official/Solutions/Krt-async-stuck/krtasync.playbook:157:(5) error recovery ignores input until this point: }
load complete (2 errors)